draft-irtf-cfrg-opaque.md

```diff
@@ -1828,6 +1828,10 @@ applications can use to control OPAQUE:
   implement this mitigation SHOULD use the same configuration information (such as
   the oprf_seed) for all clients; see {{preventing-client-enumeration}}. In settings
   where this attack is not a concern, servers may choose to not support this functionality.
+- Handling password changes: In the event of a password change, the client and server SHOULD
```

Comment:
```suggestion
- Handling password changes: In the event of a password change, the client and server can
```

Reply:
I don't think we need any normative language here. Applications can do whatever they want here.